game/static/game/css/auth.css

```
@@ -237,3 +237,68 @@ body {
+.paste-block-msg {
+    font-size: 0.75rem;
+    color: #ffb4b4;
+    padding: 6px 8px;
+    margin-top: 4px;
+    background: rgba(255, 107, 107, 0.1);
+    border-left: 2px solid #ff6b6b;
+    border-radius: 4px;
+    animation: fadeOut 0.3s ease 2.7s forwards;
+}
+
+@keyframes fadeOut {
```

Stylelint (17.11.0)

[error] Expected keyframe name "fadeOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)